scripts/Dockerfile.bottlecap.build

```diff
@@ -19,7 +19,7 @@ RUN mkdir -p /tmp/dd
 COPY ./bottlecap/src /tmp/dd/bottlecap/src
 COPY ./bottlecap/Cargo.toml /tmp/dd/bottlecap/Cargo.toml
 COPY ./bottlecap/Cargo.lock /tmp/dd/bottlecap/Cargo.lock
-ENV RUSTFLAGS="-C panic=abort -Zlocation-detail=none"
+ENV RUSTFLAGS="-C panic=abort -Zlocation-detail=none --cfg tokio_unstable"
```

Comment:
Removing this for prod, we should use it to keep collecting metrics in the future dev